llvm/test/Transforms/LoopVectorize/cost-model-no-assert.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt < %s -passes="loop-vectorize"
```

Comment:
Don't need the quotes. Also should use file check for something on the output. Alternatively, since this was an assertion in the cost model, can you get this down to one operation following along with test/Analysis/CostModel tests?